src/graph.rs

```diff
@@ -11,6 +11,7 @@ pub enum SpaceType {
     Food,
     Hazard,
     Occupied,
```

Comment:
Is this just enemy snake body now?

Reply:
or our snake body, i dont think it really makes a difference which body it is at this point